blueprint_gdal.Dockerfile

```
@@ -305,6 +307,74 @@ RUN \
     # cleanup
     rm -rf /tmp/*;
 
+
+# -----------------------------------------------------------------------------
+# GDAL build wheel
+# -----------------------------------------------------------------------------
+FROM setup as wheel
+
+# version argument
+ARG PYTHON_VERSION=3.9
+ARG NUMPY_VERSION=1.22.3
+
+# wheel directory
+ENV WHEEL_DIR="${STAGING_DIR}/usr/local/share/just/wheels"
+
+# local dependencies to /usr/local
+COPY --from=library ${STAGING_DIR}/usr/local /usr/local
+
+# build wheels
+RUN mkdir -p "${WHEEL_DIR}"; \
+    # SWIG directory
+    SWIG_DIR="$(find . -type d -name 'swig' | head -n 1)"; \
+    #
+    # test for "use_2to3" in setup.py which requires older setuptools
+    # https://github.com/OSGeo/gdal/issues/4467#issuecomment-916676916
+    if grep -q 'use_2to3' "${SWIG_DIR}/python/setup.py"; then \
+        SETUPTOOLS_DEP="setuptools<58"; \
+    fi; \
+    #
+    # python flavor
+    if [[ "${PYTHON_VERSION}" == 3.7* ]]; then \
+      PYNAME='cp37-cp37m'; \
+    elif [[ "${PYTHON_VERSION}" == 3.8* ]]; then \
+      PYNAME='cp38-cp38'; \
+    elif [[ "${PYTHON_VERSION}" == 3.9* ]]; then \
+      PYNAME='cp39-cp39'; \
+    elif [[ "${PYTHON_VERSION}" == 3.10* ]]; then \
+      PYNAME='cp310-cp310'; \
+    else \
+      echo "Unrecognized PYTHON_VERSION=${PYTHON_VERSION}" >&2; \
+      exit 1; \
+    fi; \
+    PYBIN="/opt/python/${PYNAME}/bin"; \
```

In the past, I've used the following:

```
function python_dir()
{
  a=$(echo ${1}* | sed -E 's|(.)\.([^.]*).*|\1\2|')
  echo /opt/python/cp${a}*
}
```

So maybe this will work?

```suggestion
  PYBIN=$(pyname=$(echo ${PYTHON_VERSION} | sed -E 's|(.)\.([^.]*).*|\1\2|'); echo /opt/python/cp${pyname}*/bin)
```

---

so changed.